inf-clojure.el

```diff
@@ -770,6 +809,77 @@ Return the number of nested sexp the point was over or after."
   (setq-local eldoc-documentation-function #'inf-clojure-eldoc)
   (apply #'eldoc-add-command inf-clojure-extra-eldoc-commands))
 
-(provide 'inf-clojure)
+;;;; Lumo
+;;;; ====
 
+(defgroup lumo nil
+  "Run an external Lumo process (REPL) in an Emacs buffer."
+  :group 'inf-clojure)
+
+;; AR - TODO Alternatively you can specify a command string that will be called,
+;; which should return a string.
+(defcustom inf-clojure-lumo-classpath-generator "cp"
```

Comment:
This name looks like it is arbitrary, but it is kind of standard to call it something like this.